main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -75,6 +75,8 @@
     public static final int SHOW_WP_THRESHOLD_MAX = 200;
     private static final int BROUTER_THRESHOLD_DEFAULT = 10;
     public static final int BROUTER_THRESHOLD_MAX = 120;
+    public static final int OCMAXDISTANCE_DEFAULT = 200;    // km
```

Comment:
Apart of being OC-specific (mentioned before), can we use `OC_MAX_DISTANCE_KM_DEFAULT` or something like this to remove this comment and make the units explicit in the name of the constant?